synapse/event_auth.py

```diff
@@ -324,6 +310,41 @@ def _check_size_limits(event: "EventBase") -> None:
         raise EventSizeError("event too large")
 
 
+def _check_create(event: "EventBase") -> None:
+    """Implementation of the auth rules for m.room.create events
+
+    Args:
+        event:  The event to be checked
```

Comment:
```suggestion
        event: The `m.room.create` event to be checked
```
picky, but if we assert it's one of those, it should be absolutely clear that's what we're expecting